backtrace-library/src/main/java/backtraceio/library/base/BacktraceBase.java

```diff
@@ -219,6 +386,23 @@ public void send(BacktraceReport report) {
      * @param report current BacktraceReport
      */
     public void send(BacktraceReport report, final OnServerResponseEventListener callback) {
+        if (backtraceBreadcrumbs != null && backtraceBreadcrumbs.isBreadcrumbsEnabled()) {
+            File breadcrumbLogFilesDir = new File(backtraceBreadcrumbs.getBreadcrumbLogDirectory());
+
+            File[] breadcrumbLogFiles = breadcrumbLogFilesDir.listFiles();
+
+            if (breadcrumbLogFiles.length != 1) {
```

Comment:
what if breadcrumbLogFiles.length will be zero, is it ok or impossible?

Reply:
Outdated. It was pretty confusing.